code/game/machinery/clonescanner.dm

```diff
@@ -0,0 +1,230 @@
+/// A limb that's not missing with no damages or flags.
+#define HEALTHY_LIMB list(0, 0, 0, FALSE)
+/// As above, but for organs.
+#define HEALTHY_ORGAN list(0, 0, FALSE)
+
+/// A datum to store the information gained by scanning a patient OR the fixes to be made to their body.
+/datum/cloning_data
+	/// The patient's name.
+	var/name
+	/// A reference to the patient's mind.
+	var/mindUID
+
+	/// The patient's external organs (limbs) and their data, stored as an associated list of lists.
+	/// List format: limb = list(brute, burn, status, missing, name, max damage)
+	var/list/limbs = list()
+
+	/// The patient's internal organs and their data, stored as an associated list of lists.
+	/// List format: organ = list(damage, status, missing, name, max damage, organ tag)
+	var/list/organs = list()
+
+	/// The patient's DNA
+	var/datum/dna/genetic_info
+
+//this is mostly an example
+/datum/cloning_data/healthy
+
+	limbs = list(
+		"head"   = HEALTHY_LIMB,
+		"torso"  = HEALTHY_LIMB,
+		"groin"  = HEALTHY_LIMB,
+		"r_arm"  = HEALTHY_LIMB,
+		"r_hand" = HEALTHY_LIMB,
+		"l_arm"  = HEALTHY_LIMB,
+		"l_hand" = HEALTHY_LIMB,
+		"r_leg"  = HEALTHY_LIMB,
+		"r_foot" = HEALTHY_LIMB,
+		"l_leg"  = HEALTHY_LIMB,
+		"l_foot" = HEALTHY_LIMB
+	)
+
+	organs = list(
+		"heart"    = HEALTHY_ORGAN,
+		"lungs"    = HEALTHY_ORGAN,
+		"liver"    = HEALTHY_ORGAN,
+		"kidneys"  = HEALTHY_ORGAN,
+		"brain"    = HEALTHY_ORGAN,
+		"appendix" = HEALTHY_ORGAN,
+		"eyes"     = HEALTHY_ORGAN
+	)
+
+#define SCANNER_UNCLONEABLE_SPECIES "uncloneable"
+#define SCANNER_HUSKED "husked"
+#define SCANNER_ABSORBED "absorbed"
+#define SCANNER_NO_SOUL "soulless"
+#define SCANNER_BRAIN_ISSUE "suicide or missing brain"
+#define SCANNER_MISC "miscellanious"
+#define SCANNER_SUCCESSFUL "successful"
```

Comment:
If these are used in multiple files, it might be good to move these to a cloning defines file or something